### PR TITLE
Fix: [v3] Fix test failures on chat v4

### PR DESCRIPTION
**Pull request by HoonBaek**
## Description Of Changes

* Change the messageListParams property names to fit chat v4
  * messageType to messageTypeFilter
  * customTypes to customTypesFilter
  * senderUserIds to senderUserIdsFilter
* Do not use optional chaining in null check with !== when it's used under lines
* Remove unused importing modules in the ParticipantUI component
* Install
  * `eslint-plugin-babel`
  * `@typescript-eslint/parser`
  * `@typescript-eslint/eslint-plugin`
* Change eslintrc
  * "no-unused-expressions": "off"
  * "@typescript-eslint/no-unused-expressions": ["error"]
* Fix lint error
  * Do not use optional chaining for methods in js file. so we can use it after translating to ts

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [x] Bugfix
- [ ] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [ ] Improvement (refactor code)

### CODE REVIEW

**Comment by HoonBaek** [comment]
@sravan-s Now it seems good to review!

File: package.json [hunk 1]
```diff
@@ -121,7 +122,7 @@
     "stylelint-config-sass-guidelines": "^7.0.0",
     "ts-loader": "^7.0.5",
     "tslib": "^2.0.0",
-    "typescript": "^4.3.5"
+    "typescript": "^4.7.4"
```
Comment (sravan-s) [inline package.json:125]: 👍

File: src/smart-components/Channel/components/ChannelHeader/utils.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const getChannelTitle = (
   if (!channel?.name && !channel?.members) {
     return LABEL_STRING_SET.NO_TITLE;
   }
-  if (channel?.name !== 'Group Channel') {
+  if (channel.name && channel.name !== 'Group Channel') {
```
Comment (sravan-s) [inline src/smart-components/Channel/components/ChannelHeader/utils.ts:13]: There is a chance of channel being undefined

Comment (HoonBaek) [reply src/smart-components/Channel/components/ChannelHeader/utils.ts:13]: So your suggestion is?
```
channel?.name && channel.name !== 'Group Channel'
```

**Review by sravan-s — APPROVED** [review]
Left a minor comment

**Pull request merged**